lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -41,6 +41,25 @@ destroy:
   ios_xr:
     set_value: 'no vrf <vrf>'
 
+mhost_ipv4:
```

Comment:
Agree with Mike and Charles. A second question - is 'mhost' a sufficiently specific name for this property, or should it really be 'mhost_default_interface'?